desktop.css

```diff
@@ -0,0 +1,106 @@
+@media screen and (min-width: 768px) {
+  #hamburger {
+    display: none;
+  }
```

Comment:
Your project is looking good. :+1:  The only issue I have is that the design does not conform to the desktop figma design between screen width of 768px and 1041px. Check the images below :point_down:  for illustration. Kindly ensure that it conforms to the dimensions of the desktop figma design for all screen size greater that 768px as stated in the project requirements. For your project to be responsive it has to conform to the figma designs in all screen sizes

**Headline Section**

Your design             |  Figma design
:-------------------------:|:-------------------------:
![Screenshot from 2022-09-30 15-26-02](https://user-images.githubusercontent.com/65233860/193304892-2498b9fc-b660-4924-88b3-e55a8c9952a7.png)  |  ![Screenshot from 2022-09-30 15-27-42](https://user-images.githubusercontent.com/65233860/193304955-0837e1ab-ed49-49d8-8e9f-56d69efd8b56.png)

Your design             |  Figma design
:-------------------------:|:-------------------------:
![Screenshot from 2022-09-30 15-26-17](https://user-images.githubusercontent.com/65233860/193305036-936eb02d-3c59-4a56-b3aa-08c935a5ae63.png)  |  ![Screenshot from 2022-09-30 15-27-57](https://user-images.githubusercontent.com/65233860/193305094-99a925c0-8b53-49ad-aa78-7bb646742b53.png)